examples/ecmascript/es5.ohm

```diff
@@ -338,18 +328,13 @@ ES5 {
                   | "!"    UnaryExpression  -- lnot
                   | PostfixExpression
 
-  MultiplicativeExpression = MultiplicativeExpression "*" UnaryExpression -- mul
-                           | MultiplicativeExpression "/" UnaryExpression -- div
-                           | MultiplicativeExpression "%" UnaryExpression -- mod
+  MultiplicativeExpression = MultiplicativeExpression ("*"|"/"|"%") UnaryExpression -- mul
                            | UnaryExpression
 
-  AdditiveExpression = AdditiveExpression "+" MultiplicativeExpression -- add
-		     | AdditiveExpression "-" MultiplicativeExpression -- sub
-		     | MultiplicativeExpression
+  AdditiveExpression = AdditiveExpression ("+"|"-") MultiplicativeExpression -- add
```

Comment:
With cases like this, I've never known what to use for the inline rule name. `AdditiveExpression_add` just seems weird. But it just occurred to me -- what about "rec"?